cmd/admin-rpc-server.go

```diff
@@ -105,6 +111,23 @@ func (s *adminCmd) ReInitDisks(args *AuthRPCArgs, reply *AuthRPCReply) error {
 	return nil
 }
 
+// Uptime - returns the time when object layer was initialized on this server.
+func (s *adminCmd) Uptime(args *AuthRPCArgs, reply *UptimeReply) error {
+	if err := args.IsAuthenticated(); err != nil {
+		return err
+	}
+
+	if globalBootTime.IsZero() {
+		return errServerNotInitialized
+	}
+
+	*reply = UptimeReply{
+		Uptime: time.Now().UTC().Sub(globalBootTime),
```

Comment:
Can you add a notice that this is sometimes buggy but will be automatically fixed in go 1.9 ? golang/go#12914